sp800-63-3/cover.md

```diff
@@ -168,31 +168,31 @@ The terms "CAN" and "CANNOT" indicate a possibility and capability, whether mate
 
-In today's environment, an organization's identity solution need not be a monolith, where one system or vendor provides all functionality. The market for identity services is componentized, allowing organizations and agencies to employ standards-based, pluggable identity solutions based on mission need. As such, SP 800-63 has been split into a suite of documents. RPs are required to use SP 800-63, however the remaining volumes may be used independently or in an integrated fashion depending on the component service(s) an agency requires.
+In today's environment, an organization's identity solution need not be a monolith, where one system or vendor provides all functionality. The market for identity services is componentized, allowing organizations and agencies to employ standards-based, pluggable identity solutions based on mission need. As such, SP 800-63 has been split into a suite of documents. The suite as a whole is referred to as "the guidelines," with the individual documents referred to as "volumes." RPs are required to use SP 800-63; the remaining volumes may be used independently or in an integrated fashion, depending on the component service(s) an agency requires.
 
-Each document has adopted verbs that are internationally recognized in standards organizations as normative and requirements-based. When used in a normative statement in this publication, they are CAPITALIZED for ease of identification. For example, the use of SHALL is used to denote a mandatory requirement, while the use of SHOULD refers to a technique, technology, or process that is recommended but not mandatory. For more details on the definitions of these terms see the [Requirements Notation and Conventions](#notation) at the beginning of each document.
+Each volume has adopted verbs that are internationally recognized in standards organizations as normative and requirements-based. When used in a normative statement in these guidelines, they are CAPITALIZED for ease of identification. For example, SHALL is used to denote a mandatory requirement, while SHOULD refers to a technique, technology, or process that is recommended but not mandatory. For more details on the definitions of these terms see the [Requirements Notation and Conventions](#notation) at the beginning of each document.
 
-These documents may inform - but do not restrict or constrain - the development or use of standards for application outside the federal government, such as e-commerce transactions.
+These documents may inform – but do not restrict or constrain - the development or use of standards for application outside the federal government, such as e-commerce transactions.
```

Comment:
Had trouble at first creating em-dashes in Haroop. Will update as I come across them.